frontend/src/hooks/api/relayNumber.ts

```diff
@@ -37,10 +46,23 @@ export function useRelayNumber(): SWRResponse<RelayNumberData, unknown> & {
     relayNumber.mutate();
     return response;
   };
+  /**
+   * Set the forwarding state
+   */
+  const setForwardingState: PhoneNumberUpdateRelayNumberForwardingStateFn =
```

Comment:
suggestion (blocking): We might want to come up with a less confusing name for the function @codemist told you I'm terrible with names. `PhoneNumberUpdateRelayNumberForwardingStateFn` feels confusing as it has both PhoneNumber and RelayNumber in the name.

Reply:
Haha! Just updated the name to `UpdateForwardingToPhone`, hopefully that's somewhat clearer lol.